service/controller/clusterapi/v31/resource/tccpazs/create_test.go

```diff
@@ -641,3 +858,7 @@ func mustParseCIDR(v string) net.IPNet {
 	}
 	return *n
 }
+
+func toNetPtr(n net.IPNet) *net.IPNet {
+	return &n
+}
```

Comment:
We could extend https://github.com/giantswarm/to with this.

Reply:
I was thinking about that, but I'm not sure how often this would be needed. The conversion is slightly oxymoron because `net.ParseCIDR()` actually returns a pointer, but due to use of value elsewhere (that does make sense there), non-pointer is more useful in this test file and it's only few cases where that's needed.